src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? (
+            <p>
+              { `${partnerName} is my ` }
+              { sex === 'm' ? 'wife' : 'husband' }
+            </p>
+          ) : (
+            <p>I am not married</p>
+          )
+        }
```

Comment:
This is really hard to read
1. Let's calculate this logic outside of JSX in some variable `sex === 'm' ? 'wife' : 'husband'`
2. You don't need these inner `<p>`, just
```
<p className="Person__partner">
  {isMarried
    ? `${partnerName} is my ${calculatedLogic}`
    :  'I am not married'
  }
</p>